manifest/services.py

```diff
@@ -62,14 +66,27 @@ def flatten(self, obj: StrEnum, data):
 jsonpickle.handlers.registry.register(ManifestStatus, JsonEnumHandler)
 
 
-class GcpBucketService(object):
-    def __init__(self, bucket_name=None, path=None):
+class GcpBucketService:
```

Comment:
This helper service would also benefit from some extended explanations on what every method does